src/main/java/com/gd/spring/Controller.java

```diff
@@ -0,0 +1,13 @@
+package com.gd.spring;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Controller {
+
+    @GetMapping("/")
```

Comment:
remove `"/"` from parameter, always use just `@GetMapping`, if you don't need to add anything
this is because if you have for example `/pets/` endpoint and if you go to `localhost:8080/pets` it will return you 404 not found - in such case the only way is to always explicitly add `/` to the path
but at the same time you will be able to visit `/pets/` if you don't have this sign at you `GetMapping` path